tests/tensor_base/test_augmented_updates.py

```diff
@@ -0,0 +1,111 @@
+from numpy.testing import assert_allclose
+import mygrad as mg
+import numpy as np
+
+
+# TODO: These tests need to be modified to check that tensor-identity
+# is preserved by the augmented update once #280 is resolved
+# https://github.com/rsokl/MyGrad/pull/280/
+def test_augmented_multiply():
+    a = np.arange(9.0).reshape(3, 3)
+    expected_grad = np.where(a < 4, 2, 1)
+    a[a < 4] *= 2
```

Comment:
Can you swap lines 11 and 12 so it's clear why the gradient should be expected after you do the operation rather than before?

Reply:
`a` get's mutated in line 12, which is why I need to order it the way I did.

Reply:
Ha, right.